### PR TITLE
implemented volume filter

### PR DESCRIPTION
**Pull request by matthias-luger**
filter style change

### CODE REVIEW

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-359.flou21.de

File: src/components/Flipper/Flipper.tsx [hunk 1]
```diff
@@ -338,28 +341,31 @@ function Flipper() {
...
+                                <Form.Label htmlFor="autoScrollCheckbox" style={{ marginRight: "10px" }}>Auto-Scroll?</Form.Label>
+                                <Form.Check inline id="autoScrollCheckbox" checked={autoscroll} onChange={(e) => { _setAutoScroll(e.target.checked) }} type="checkbox" />
+                            </Form.Group>
+                            <Form.Group>
+                                <div style={{ display: "contents", cursor: "pointer", marginRight: "10px" }} onClick={clearFlips}>
+                                    <Form.Label>Clear flips!</Form.Label>
+                                    <DeleteIcon color="error" />
                                 </div>
                             </Form.Group>
                             <Form.Group onClick={onArrowRightClick}>
```
Comment (Ekwav) [inline src/components/Flipper/Flipper.tsx:356]: This isn't the correct click handler for this.
Please display a "Coming Soon" Tooltip for this element and remove the onClick for now

File: src/components/Flipper/FlipperFilter/FlipperFilter.tsx [hunk 2]
```diff
@@ -124,13 +134,17 @@ function FlipperFilter(props: Props) {
...
+    const numberFilters = <div style={{ display: "flex", alignContent: "center", justifyContent: "flex-start" }}>
+        <Form.Group className="filterTextfield">
+            <Form.Label className="flipper-filter-formfield-label">Min. Profit:</Form.Label>
             <Form.Control onChange={onMinProfitChange} className="flipper-filter-formfield" type="number" step={5000} disabled={!props.isLoggedIn && !freeLoginFilters} />
         </Form.Group>
-        <Form.Group style={{ width: "45%", display: "inline-block", marginLeft: "5%" }}>
-            <Form.Label className="flipper-filter-formfield-label">Max Cost:</Form.Label>
+        <Form.Group className="filterTextfield">
+            <Form.Label className="flipper-filter-formfield-label">Min. Volume:</Form.Label>
+            <Form.Control onChange={onMinVolumeChange} className="flipper-filter-formfield" type="number" max={60} disabled={!props.isLoggedIn && !freeLoginFilters} />
```
Comment (Ekwav) [inline src/components/Flipper/FlipperFilter/FlipperFilter.tsx:144]: could negative numbers be used to invert the filter to "maxVolume" ?

**Review by Ekwav — CHANGES_REQUESTED** [review]
Looks dope, greatly reduces the height of the options

**Comment by Flou21** [comment]
[CI] created a pr environment: https://pr-env-sky-359.flou21.de

**Review by Ekwav — APPROVED** [review]

**Pull request merged**